__tests__/App.test.js

```diff
@@ -3,9 +3,17 @@ import enzyme from 'enzyme';
 import App from '../client/src/components/App.jsx';
 
 describe('<App />', () => {
-  it('renders 1 <App /> component', () => {
+  it('should render one <App /> component', () => {
     const component = enzyme.shallow(<App />);
     expect(component).toHaveLength(1);
   });
+
+  it('should call setRelatedProducts when component mounts', () => {
```

Comment:
change to: should call setRelatedProducts when App component updates on mount

Reply:
@TomGeshury I'm not checking if the component is being updated I'm checking to see if the component calls a particular method when the component first mounts.